tests/output/bug76136.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+Bug #76136: stream_socket_get_name should enclose IPv6 in brackets
+--FILE--
+<?php
+$server = stream_socket_server("tcp://[::1]:1337/");
+echo stream_socket_get_name($server, false).PHP_EOL;
+$server = stream_socket_server("tcp://127.0.0.1:1337/");
+echo stream_socket_get_name($server, false);
+--EXPECTF--
```

Comment:
Missing `?>`.

Also, please use `EXPECT`, is simpler than `EXPECTF` :+1:

Reply:
Done.